library/src/scala/quoted/Quotes.scala

```diff
@@ -3807,6 +3868,14 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
         /** Case class or case object children of a sealed trait or cases of an `enum`. */
         def children: List[Symbol]
+
+        /** Type reference to the symbol */
```

Comment:
Maybe mention that this is as seen from the owner:
```suggestion
        /** Type reference to the symbol usable in the scope of its owner. */
```